pkgs/tools/misc/translate-toolkit/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchFromGitHub, python3Packages, makeWrapper }:
+
+python3Packages.buildPythonApplication rec{
```

Comment:
We don't store python applications here, as we we build `pythonPackages` against multiple python versions.
Please change to `buildPythonLibrary` here.